iceoryx_hoofs/include/iceoryx_hoofs/internal/cxx/optional.inl

```diff
@@ -90,6 +94,7 @@ inline optional<T>::optional(in_place_t, Targs&&... args) noexcept
     construct_value(std::forward<Targs>(args)...);
 }
 
+// AXIVION Next Construct AutosarC++19_03-A13.3.1 : It's safe and constrained by SFINAE
```

Comment:
I think this is a false positive. Not sure SFINAE is in play here, if there is no overload resolution set where ill-formed overloads are eliminated. I cannot see an overload with a forwarding reference (this would require `optional<T>` to be a template parameter).

@FerdinandSpitzschnueffler Maybe I am wrong here. Could you please double check?

Reply:
@MatthiasKillat It is false positive, and is now been disabled for these scenarios. i.e "overloaded_fref":

/cc @FerdinandSpitzschnueffler